OpenGL/OpenGL/src/Camera.cpp

```diff
@@ -6,10 +6,12 @@
 #include <iostream>
 
 Camera::Camera(unsigned int width, unsigned int height, const glm::vec3 &position,
-               const glm::vec3 &center, Projection projType):
- position(position), center(center), fovY(45.f), near_(0.1f), far_(100.f),
+               const glm::vec3 &rotation, Projection projType):
+ position(position), center(rotation), fovY(45.f), near_(0.1f), far_(100.f),
```

Comment:
If center is now a rotation then it should be renamed and also make sure that the rest of Camera's code is updated to work with rotation (I believe you broke the rest of Camera)

Reply:
if some of camera's code is broken but not used anymore, we can just remove it